vector/src/main/res/values/strings.xml

```diff
@@ -1584,9 +1584,9 @@
     <string name="room_list_catchup_empty_title">You’re all caught up!</string>
     <string name="room_list_catchup_empty_body">You have no more unread messages</string>
     <string name="room_list_people_empty_title">Conversations</string>
-    <string name="room_list_people_empty_body">Your direct message conversations will be displayed here. Tap the + bottom right to start some.</string>
+    <string name="room_list_people_empty_body">Your direct message conversations will be displayed here. Tap the + at the bottom right to start some.</string>
     <string name="room_list_rooms_empty_title">Rooms</string>
-    <string name="room_list_rooms_empty_body">Your rooms will be displayed here. Tap the + bottom right to find existing ones or start some of your own.</string>
+    <string name="room_list_rooms_empty_body">Your rooms will be displayed here. Tap the + at the bottom right to find existing ones or start some of your own.</string>
```

Comment:
Note to myself: add a comment above this 2 strings to let translators knows that on RTL languages the button will be at bottom left.

There are some mistake currently about that:

<img width="1299" alt="image" src="https://user-images.githubusercontent.com/3940906/170230440-7379cf23-872d-479a-b43f-3422a154b4d2.png">

Since this string will be updated, the translators will have the opportunity to fix that.